packages/host/app/components/ai-assistant/card-picker/index.gts

```diff
@@ -127,9 +148,32 @@ export default class AiAssistantCardPicker extends Component<Signature> {
       .is-autoattached {
         border-style: dashed;
       }
+      .view-all {
+        cursor: pointer;
```

Comment:
Hm, isn't this pill a button tag, which should have pointer enabled by default?

Reply:
No it isn't, I reused `Pill` component for this.